packages/iov-keycontrol/src/prehashing.ts

```diff
@@ -1,14 +1,14 @@
 import { PrehashType, SignableBytes } from "@iov/bcp-types";
 import { Sha256, Sha512 } from "@iov/crypto";
 
-export function prehash(bytes: SignableBytes, type: PrehashType): SignableBytes {
+export function prehash(bytes: SignableBytes, type: PrehashType): Uint8Array {
```

Comment:
Maybe add more info rather than losing it?
Like `PrehashedSignableBytes`
Then we could enforce that prehash was run before signing later on.